### PR TITLE
reduce repeated memory allocation

### PR DESCRIPTION
**Pull request by kongliang1**

### CODE REVIEW

**Comment by tanvir-retailai** [comment]
LGTM

File: helper/redisx/lua/scripts.go
Comment (zhongdalu-trechina) [inline helper/redisx/lua/scripts.go:1]: better put this file into `redisx` folder instead of `lua` folder.

Comment (kongliang1) [reply helper/redisx/lua/scripts.go:1]: done.

**Review by zhongdalu-trechina — APPROVED** [review]
LGTM

**Pull request merged**